app/jobs/ingest_local_file_job.rb

```diff
@@ -20,12 +20,13 @@ def run
     path = File.join(directory, filename)
 
     generic_file.label = File.basename(filename)
-    generic_file.add_file(File.open(path), 'content', generic_file.label)
+    generic_file.add_file(File.open(path), path: 'content', original_name: generic_file.label, mime_type: mime_type(filename))
     generic_file.record_version_committer(user)
     generic_file.save!
```

Comment:
Yes we should (See https://github.com/projecthydra/sufia/blob/latest_hydra_head/app/jobs/ingest_local_file_job.rb#L15)  I took a stab at that, but I was getting too many unrelated issues, so I backed it out.